src/test/java/hudson/plugins/emailext/plugins/content/FailedTestsContentTest.java

```diff
@@ -68,7 +68,6 @@ public void testGetContent_withWorkspaceAndNoTestsRanShouldGiveAMeaningfulMessag
     public void testGetContent_whenAllTestsPassedShouldGiveMeaningfulMessage()
             throws Exception {
         AbstractTestResultAction testResults = mock( AbstractTestResultAction.class );
-        when( testResults.getFailCount() ).thenReturn( 0 );
```

Comment:
Why is this test being modified?

Reply:
`testResult.getFailCount()` is not being called anymore as the count is dependent on the filtered tests. I guess I can let it be and update the count after filtering so that other tests remain unmodified.